### PR TITLE
replace exif with exifr

### PR DESCRIPTION
**Pull request by taymoork2**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/helper-image/src/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 /* global Uint8Array, FileReader */
 
 const {Buffer} = require('safe-buffer');
-const {ExifImage} = require('exif');
+const {parse} = require('exifr/dist/lite.umd');
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/helper-image/src/index.js:9]: Glad we're moving away from the older `exif` package.

File: packages/node_modules/@webex/helper-image/src/index.js [hunk 2]
```diff
@@ -43,21 +43,27 @@ export function updateImageOrientation(file, options = {}) {
 * @returns {Promise<ExifImage>}
 */
 export function readExifData(file, buf) {
-  return new Promise((resolve) => {
+  return new Promise(async (resolve) => {
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/helper-image/src/index.js:46]: Not a big fan of mixing async with promises in this way, but, if it works, it works.

File: packages/node_modules/@webex/helper-image/src/index.js [hunk 3]
```diff
@@ -43,21 +43,27 @@ export function updateImageOrientation(file, options = {}) {
...
-      });
+    if (
+      file &&
+      (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')
+    ) {
+      const {
+        Orientation,
+        ExifImageHeight,
+        ExifImageWidth
+      } = await parse(buf, {translateValues: false});
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/helper-image/src/index.js:56]: in this case, if an error is thrown here, the promise returned by `readExifData` *should* reject, so the original error validation does not appear to be necessary any longer.

**Comment by brunogenaro** [comment]
@taymoork2 can you review @InteractiveTimmy requests when you have a chance?

File: packages/node_modules/@webex/helper-image/src/index.js (hunk 2 above)
Comment (taymoork2) [reply packages/node_modules/@webex/helper-image/src/index.js:46]: `await`'ing is the suggested way to grab data hence the async

**Comment by taymoork2** [comment]
@brunogenaro I think's tim's comments were just more "explanations" on my implementation rather than review comments

**Pull request merged**